src/views/Main/Schemas/Components/EntityType.js

```diff
@@ -13,7 +13,8 @@ import styles from '../styles.module.css';
 export class EntityType extends React.Component {
   static propTypes = {
     entityType: PropTypes.object,
-    idToPropertyTypes: PropTypes.object
+    idToPropertyTypes: PropTypes.object,
```

Comment:
This is fine, but it's not recommended to use `PropTypes.object`. The recommended way is to use `PropTypes.shape({})`.